### ISSUE TITLE
Move output to panel???

### ISSUE THREAD

**Issue by mlcampbe** [bnason-nf/findallinfile#4]
This extension works great but the sidebar is too narrow (by default) to see the text of the line. If I expand the sidebar then the editor side is too small. So, is there any chance you can force the output to the panel so that the full width of the window can be used to preview the found lines?

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

The original version of this extension did that, so it's definitely possible. However as far as I know it's not possible to have the panel be interactive to allow you to select results and go to the right spot in the file.

It's probably possible to add a setting to select where the output should go (side bar, panel, or both). Would that make sense to you? Or would non-interactive panel results not be useful?

Thanks,
Benbuck

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

If I can figure out how to do it, would it work for you if I could get tool tip text for the whole line to pop up as you hover over each result (if that makes sense)?

Thanks,
Benbuck

**Comment by mlcampbe** [comment]
Benbuck,

I don't think it would be very helpful to not be able to click a search result and it not jump to that point in the file. Therefore in the side bar with a popup might be the best option.

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

Funny, I had forgotten that the tool tip thing already works. Can you try just hovering over the matches to see the pop up tool tips? It looks like a recent change I made has formatting that doesn't work well in the tool tip, but other than that please let me know how that works for you.

Thanks,
Benbuck

**Comment by mlcampbe** [comment]
Benbuck,

Yes actually the popup is working. It is a little slow to show up but it does work.

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

It doesn't seem slow for me. When it's slow for you, how big is the file? How many matches were found?

Thanks,
Benbuck

**Comment by mlcampbe** [comment]
Let me rephrase "slow" then. I search a file and it finds 2 references and they are shown in the side bar. When I first move my mouse on top of 1 of the entries it takes about 2 seconds for the popup to appear. If I then move to the other entry it pops up almost instantly. But, if the mouse leaves the side bar and comes back it again takes 2 seconds to popup.

**Comment by bnason-nf** [comment]
Hi @mlcampbe ,

I'm not seeing that behavior, and I'm not sure what I could do differently in the extension even if I were. If you click instead of hovering, do you see similar delays? Are the matched lines long? Can you think of anything else that might help figure this out?

Thanks,
Benbuck

**Comment by mlcampbe** [comment]
Not sure what to tell you but I am ok with it for now. If you could figure out how to move the results to an output window AND click on the line to jump to that point in the file that would be great. If not it is still a very very useful extension.

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

Alright, I'm going to close this then. Please let me know if you figure out anything about what might be making it slow.

Thanks,
Benbuck

**Comment by bnason-nf** [comment]
Hi @mlcampbe,

This is a fortunate coincidence, with the latest version of VSCode that was released today, you can drag the views from the side bar to the panel if you want:

https://code.visualstudio.com/updates/v1_46#_flexible-layout

I'm guessing that feature should completely solve this for you

Thanks,
Benbuck.

**Comment by mlcampbe** [comment]
Benbuck,

Yep works like a charm now.
